lib/Ed25519VerificationKey2020.js

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 import * as base58btc from 'base58-universal';
-import * as ed25519 from '@stablelib/ed25519';
+import ed25519 from './ed25519';
```

Comment:
Always include the extension if it is a file and not a module that is being imported:

```suggestion
import ed25519 from './ed25519.js';
```